app/src/main/java/com/mapbox/maps/testapp/examples/snapshotter/LocalStyleMapSnapshotterActivity.kt

```diff
@@ -63,30 +62,21 @@ class LocalStyleMapSnapshotterActivity : AppCompatActivity(), Snapshotter.Snapsh
         }
       """.trimIndent()
     )
-    mapSnapshotter.start(this)
-  }
-
-  override fun onStyleLoaded(style: Style) {
-    Logger.i(TAG, "OnStyleLoaded: ${style.styleURI}")
-  }
-
-  override fun onSnapshotCreated(snapshot: Expected<MapSnapshotInterface?, String?>) {
-    if (snapshot.isValue) {
-      val mapSnapshot = snapshot.value as MapSnapshotInterface
-      Logger.e(TAG, "Result: ${mapSnapshot.image().data.size}")
+    mapSnapshotter.start {
+      it?.let { mapSnapshot ->
+        Logger.e(TAG, "Result: ${mapSnapshot.image().data.size}")
```

Comment:
nit log can be removed.

Reply:
I did not touch that logic, I guess author of that example did it intentionally.